lib/tether/ops_darwin.go

```diff
@@ -96,24 +95,58 @@ func (t *BaseOperations) Cleanup() error {
 	return nil
 }
 
-// Need to put this here because Windows does not
-// support SysProcAttr.Credential
-func getUserSysProcAttr(uname string) *syscall.SysProcAttr {
```

Comment:
Do we need the duplicate code in ops_darwin? If no, let's just return nil like the windows version.  If yes, we could move this function to ops_unix.go and it'll be shared by linux+darwin.